frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -112,6 +117,43 @@ function FileManagerMenu:setUpdateItemTable()
     end
 
     -- tools tab
+    if Device.isKobo() then
```

Comment:
Yeah, I first started trying to move this part into device specific code. Then I ended up not doing it because I kind of want to avoid polluting device module with something we will throw away in the future soon. I still haven't decided on this one yet.

> The problem is that the menu would have to be rewritten lest menu item
order is completely haphazard. That's already a bit of an issue.

@Frenzie , You mean it's hard to control the order of menu items?